_posts/2016-06-22-hello-beercraft.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: post
+title: Maku brewing and Solita's passion pale ale
+author: Rinorragi
+excerpt: Solita ordered passion pale ale from Maku brewing and got introduction to their craftmenship
+tags: 
+- Beercraft
+---
+Solita wanted to have their own beer. The process was like a software project. First we tried to figure out what we want. We had our internal voting for the taste and name for the beer and "Intohimo" passion pale ale won the contest. After knowing what we need we went and sent multiple offers to different breweries. Among the offers we selected Maku brewing and decided with them that the receipt needs to be tested first. Maku brewing created 20l patch of the receipt they invented for us and at 21.6.2016 we got chance to taste the beer and make adjustments to it. All started with bus trip from Solita's office at Helsinki to Tuusula. We had great time already in the bus in discussion of projects and expectations of the upcoming beer. 
+
+## Maku brewing
+
+[Maku brewing](http://www.makubrewing.com/) is few years old finnish craft beer brewery. They are making great beers. Their IPA is most likely the best finnish IPA and it got some awards as well. Their amber ale is also great and it has been awarder as well. 
+
+After fitting 20 nerds into a bus we got introductionary into their brewery by Henri Parviainen who is just leaving the IT business and starting to be full-day at brewery. Their business started when few guys invested few hundred euros to 20l Braumeister home brewery system at 2014. Braumeister automates the boiling process for home brewery. After half a year they bought bigger equipments from another brewery and started to make bigger amounts of beer at the same time. Now they have 3000l boiling equipment and 6x 6000l and 4x 2000l fermentation tanks. 
+
+Here we are at the bottling chamber. It serves also as a warehouse. 
+![Tools](/img/hello-beercraft/the_people.jpg)
+
+## Contest for label 
+
+We had our internal contest for best labels. Here is a picture of winning label. 
+![Tools](/img/hello-beercraft/intohimo.png)
+
+There were plenty of great contesters but I have to admit that I was imressed by the winner. Winner created 3D model that can be found at [this website](http://dev.makarainen.org/intohimo/)
+
+Committee that chose winner said that the winner was able to combine design and techinical skills in a Solita like manner. Colleagues said that the look and feel was really fresh, it had artistic passion and everything fits!
+
+## Tasting notes
+
+Smell and taste had refreshing taste of passion fruit as we wanted. Citra and Jarrylo hops created bitterness in the end that was suitable for any summer beer. We were happy about how well passion fruit came out in both smell and taste. We had 20l test series and we were so happy that we let Maku to continue without making any changes. 
+
```

Comment:
Tämä kuulostaa siltä, että joitte 20 litraa testikaljaa ja olitte sen jälkeen tyytyväisiä :) Se voi olla toki tarkoituskin!

We were satisfied with the test series (vai test batch?) and gave Maku permission to proceed to production serias (vai batch?).